### DISCUSSION TITLE
RPi.GPIO not working

### DISCUSSION THREAD

**Discussion by lochbika** [rern/rAudio#170] [Fixed Issues] [answered]
Hi, with the latest image I figured out that the RPi.GPIO module is not working anymore. When I load it in a interactive python console I get the following error:

```
Python 3.9.1 (default, Dec 15 2020, 20:19:56) 
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import RPi.GPIO
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.9/site-packages/RPi/GPIO/__init__.py", line 23, in <module>
    from RPi._GPIO import *
  File "/usr/lib/python3.9/site-packages/RPi/_GPIO.py", line 9, in <module>
    __bootstrap__()
  File "/usr/lib/python3.9/site-packages/RPi/_GPIO.py", line 7, in __bootstrap__
    mod = importlib.util.module_from_spec(spec)
ImportError: /usr/lib/python3.9/site-packages/RPi/_GPIO.cpython-39-aarch64-linux-gnu.so: wrong ELF class: ELFCLASS64
>>> 
```

**Comment by lochbika** [comment]
Udate: manually removing the package directory `/usr/lib/python3.9/site-packages/RPi` and installing the AUR version fixed the issue. @rern you might want to update your custom package in the rOS repository.

Cheers

**Comment by rern** [comment, marked as answer]
GPIO Relays script has been switched from python `RPi.GPIO` to `wiringpi` package which is more efficient and easier to maintain.
